core/modules/filters/is/draft.js

```diff
@@ -19,7 +19,7 @@ exports.draft = function(source,prefix,options) {
 	var results = [];
 	if(prefix === "!") {
 		source(function(tiddler,title) {
-			if(!tiddler || !$tw.utils.hop(tiddler.fields,"draft.of")) {
+			if(!tiddler || !$tw.utils.hop(tiddler.fields,"draft.of") || (tiddler.fields["draft.of"].length === 0)) {
```

Comment:
There is a `tiddler.isDraft()` method in the core that perhaps would be worth using here.

What is the rationale for the new test for the length of the draft not being zero? Is there a situation where a zero length `draft.of` field is likely to occur?